### ISSUE TITLE
Error if player has no playerex:level attribute

### ISSUE THREAD

**Issue by Chrissdroid** [CleverNucleus/PlayerEx#20]
everything is in the title make the client disconnect if he type `/playerex levelup <playerName>` prevent also the player from lvl up but consume the levels. 

```log
java.lang.IllegalArgumentException: Can't find attribute playerex:level
	at net.minecraft.class_5132.method_26865(class_5132.java:22) ~[intermediary-server.jar:?]
	at net.minecraft.class_5132.method_26862(class_5132.java:28) ~[intermediary-server.jar:?]
	at net.minecraft.class_5131.method_26852(class_5131.java:62) ~[intermediary-server.jar:?]
	at com.github.clevernucleus.playerex.handler.CommandsHandler.levelUpAlt(CommandsHandler.java:130) ~[playerex-3.0.2-1.17.1.jar:?]
	at com.github.clevernucleus.playerex.handler.CommandsHandler$$Lambda$6835/0x00000000efda71a0.run(Unknown Source) ~[?:?]
	at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:263) ~[intermediary-server.jar:?]
	at net.minecraft.class_2170.redirect$zjj000$performCommand(class_2170.java:547) [intermediary-server.jar:?]
	at net.minecraft.class_2170.method_9249(class_2170.java:251) [intermediary-server.jar:?]
	at net.minecraft.class_3244.method_14370(class_3244.java:1222) [intermediary-server.jar:?]
	at net.minecraft.class_3244.method_31286(class_3244.java:1206) [intermediary-server.jar:?]
	at net.minecraft.class_3244.method_12048(class_3244.java:1189) [intermediary-server.jar:?]
	at net.minecraft.class_2797.method_12115(class_2797.java:30) [intermediary-server.jar:?]
	at net.minecraft.class_2797.method_11054(class_2797.java:7) [intermediary-server.jar:?]
	at net.minecraft.class_2600.method_11072(class_2600.java:21) [intermediary-server.jar:?]
	at net.minecraft.class_2600$$Lambda$9149/0x00000000fc8079a8.run(Unknown Source) [intermediary-server.jar:?]
	at net.minecraft.class_3738.run(class_3738.java:18) [intermediary-server.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:151) [intermediary-server.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) [intermediary-server.jar:?]
	at net.minecraft.server.MinecraftServer.method_24306(MinecraftServer.java:788) [intermediary-server.jar:?]
	at net.minecraft.server.MinecraftServer.method_18859(MinecraftServer.java:164) [intermediary-server.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:125) [intermediary-server.jar:?]
	at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:770) [intermediary-server.jar:?]
	at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:764) [intermediary-server.jar:?]
	at net.minecraft.class_1255.method_18857(class_1255.java:134) [intermediary-server.jar:?]
	at net.minecraft.server.MinecraftServer.method_16208(MinecraftServer.java:749) [intermediary-server.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:701) [intermediary-server.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270) [intermediary-server.jar:?]
	at net.minecraft.server.MinecraftServer$$Lambda$7778/0x00000000eff6c7b8.run(Unknown Source) [intermediary-server.jar:?]
	at java.lang.Thread.run(Thread.java:853) [?:?]
```

**Comment by CleverNucleus** [comment]
Hi there,
I'll have a fix in the next release.
Thanks.

**Comment by Chrissdroid** [comment]
Thanks for the fast response

Keep up the good work
Your mod is pretty good and integrates nicely with others scaling mods !

**Comment by CleverNucleus** [comment]
Fixed in latest release, 3.0.3.
